### PR TITLE
Live 88 participants list

### PR DESCRIPTION
**Pull request by Karolk99**

### CODE REVIEW

File: assets/js/components/ParticipantsList.tsx [hunk 1]
```diff
@@ -53,13 +48,9 @@ const ModeratorMenu = ({ username, name, isPresenter, eventChannel }: ModeratorM
...
   return (
     <Menu>
-      <MenuButton
-        as={IconButton}
-        aroa-label="Options"
-        icon={"Menu Icon"}
-        variant="outline"
-        backgroundColor="red"
-      />
+      <MenuButton ml={"auto"} aroa-label="Options">
```
Comment (Rados13) [inline assets/js/components/ParticipantsList.tsx:51]: Isn't it a typo?

File: assets/js/components/ParticipantsList.tsx [hunk 2]
```diff
@@ -79,30 +70,37 @@ const Participant = ({
...
-      {isPresenter && <b>Pres</b>}
-      <Spacer />
+    <div className="Participant">
       {isModerator ? (
+        <img src={crownIcon}></img>
+      ) : isPresenter ? (
+        <img src={starIcon}></img>
+      ) : (
+        <img src={userIcon}></img>
+      )}
```
Comment (Rados13) [inline assets/js/components/ParticipantsList.tsx:84]: Couldn't we evaluate the source before defining the component? This way we can define `img` element only once.

**Comment by Rados13** [comment]
In addition accordingly to Figma project, participants should be listed this way that presenters are on top of the list.

**Comment by Karolk99** [comment]
> senters are on top of the list.

This feature will be added in LIVE-87

**Review by Rados13 — APPROVED** [review]

**Pull request merged**